circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/conf/TableParametersConfig.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2016-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.core.conf;
+
+import java.util.Map;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "table-parameters")
+public class TableParametersConfig {
```

Comment:
bit late to the party apologies for that but Hive refers to these things as table properties: https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-listTableProperties
Maybe we use the same name will be easier to understand what it overrides.

Reply:
Yep in Hive they're properties, but on the Java Table object they're parameters. Maybe everything user facing should be properties, and internally parameters? Or do you think everything should just be properties? Thanks for the suggestion

Reply:
oh that's a good point hmm tough one. What do the others think?

Reply:
Yeah, go Hive for once again forcing us to do horrible things! I'm good with using parameters internally and properties externally to match what Hive appear to do.